src/stats.rs

```diff
@@ -0,0 +1,121 @@
+use std::collections::VecDeque;
+use std::time::{Duration, Instant};
+
+use crate::model::ComponentStats;
+
+const DEFAULT_STAT_WINDOW: Duration = Duration::from_secs(5 * 60);
```

Comment:
Is this 5 minutes?

Reply:
Yeah unfortunately Rust doesn't have a `Duration::from_mins` method